README.md

```diff
@@ -99,7 +99,8 @@ gvm.graalvm(
     ],
 )
 use_repo(gvm, "graalvm")
-register_toolchains("@graalvm//:all")
+register_toolchains("@graalvm//:jvm")
+register_toolchains("@graalvm//:sdk")
```

Comment:
`@graalvm//:all` no longer actually points to any toolchains, because they are technically `alias` targets; as such, registration will have to change to this form, which will be documented as a breaking change in the `0.10.x` release series.